packages/template-ui/src/views/Content.vue

```diff
@@ -5,8 +5,15 @@
       :title="content.title"
       :subtitle="subtitle"
     >
+      <b-row class="my-3">
+        <b-col xs="12" md="6">
```

Comment:
So with this the card occupies half of the width in large screens. I don't think this is Joana's intention but is a good start, so it's ready for merging. But please mention this implementation decision in the commit message before merging.